src/main/java/org/helpberkeley/memberdata/RestaurantTemplateParser.java

```diff
@@ -237,6 +245,22 @@ private void auditControlBlockRow(RestaurantBean bean) {
         }
     }
 
+    private void auditControlBlockFormula(RestaurantBean bean) {
+        boolean containsFormulas = false;
+        String[] formulas = bean.getFormulas();
+
+        for (String s: formulas) {
+            if (s.startsWith("=")) {
+                containsFormulas = true;
+            }
+        }
+
+        if (!containsFormulas) {
+            throw new MemberDataException("Invalid or missing Formula value found at line " + lineNumber
+                    + ". Formula values must begin with \"= or they will not import/export correctly.\n");
```

Comment:
Use Constant and MessageFormat.format

Return this a a String instead of throwing. This way it doesn't throw away any errors that were already collected in the caller.